core/src/test/java/org/infinispan/notifications/cachelistener/CacheNotifierImplTest.java

```diff
@@ -72,11 +73,19 @@ public void setUp() {
                            mock(ClusterEventManager.class));
       cl = new CacheListener();
       n.start();
-      n.addListener(cl);
+      addListener();
       ctx = new NonTxInvocationContext(null);
    }
 
-   public void testNotifyCacheEntryCreated() {
+	protected void addListener() {
+		n.addListener(cl);
+	}
+
+	protected boolean isReturnedEventValueCorrect(CacheEntryEvent ev, Object key, Object val, Type t) {
+		return val==ev.getValue() || val!=null && val.equals(ev.getValue());
```

Comment:
Spacing. Make sure you have the formatter for your editor imported [1]

[1] https://github.com/infinispan/infinispan/tree/master/ide-settings